tests/database/image/unit/test_image.py

```diff
@@ -41,13 +47,16 @@ def tearDown(self):
 
```

Comment:
Commenting here because I can't comment near the lines that aren't a part of the commit. 

Line 29: `if 'image[0] == 'image 3':`; just reversed the order for a more conventional look (variables to the left, and constants on the right)
Moreover, how about we do this
`assert image[0] == 'image 3'` I don't see a need for a for loop here. we can just access the element by it's index. 

that would be cleaner and the same thing for block starting at line 35. 

Line 41 and 42: how about `assert 'image 2' in images` or if you want to use `assertTrue` we could do `assertTrue('image 2' in images)`. 

and then in the tearDown of this test, is deleting the project only sufficient?  do we not have to delete the images first?